android/settings.gradle

```diff
@@ -39,6 +39,7 @@ project(":expo-random").projectDir = new File("../packages/expo-random/android")
 useExpoModules([
     exclude : [
         'expo-module-template',
+        'expo-in-app-purchases',
```

Comment:
So it was included in the previous version? 😮

Reply:
i think no. there is a sanity check for versioning script: https://github.com/expo/expo/blob/75c7ead1353d98508129058290b1271c6272fcf5/tools/src/Packages.ts#L206-L223
and it previously matched to the standalone `useExpoModules` which [is removed in this commit](https://github.com/expo/expo/commit/21d457592eab92917d9091a27961a413a1177e84#diff-9a50ef936e92a26017fd739f42ee0a0e7f80aec39dafc0a13f27b2548ee5e51f).